### ISSUE TITLE
[Hackathon] - Fix copyright, pre/ul styling and EPUB validation issues

### ISSUE THREAD

**Issue by richard015ar** [pressbooks/pressbooks#2470]
- [x] message prince in copyright
- [x] Validation for EPUB (Talk to Steel)
- [x] `<pre>` and `<ul>` elements are printed different in PDF

**Comment by richard015ar** [comment]
Copyright is rendered as expected, only difference is the text
`This book was produced with Pressbooks (https://pressbooks.com) and
rendered with Prince. `
which is enabled when certain global variables are present, see: https://github.com/pressbooks/pressbooks/blob/38b69cb3b9ec488dd2ea0202e35c7f929ec1519e/inc/modules/export/prince/class-pdf.php#L277

I used `dev` branch locally to make sure it is displayed in the same way than before.
For integrations probably we don't have that variable enabled.

**Comment by richard015ar** [comment]
Validation issues:
I cloned this book: https://integrations.pressbooks.network/jctestbook/

Exporting this book on integration using the current dev branch, has validation issues.
Using dev branch on my local environment I see the validation issues are:
```
2021/11/01 19:26:32 [error] 1000#1000: *571 FastCGI sent in stderr: "PHP message: Pressbooks\Modules\Export\Epub\Epub
Array
(
    [time] => Mon Nov  1 19:26:32 2021
    [user] => admin
    [site_url] => https://pressbooks.test/jctestclonedintegration
    [blog_id] => 245
    [theme] => McLuhan
)
FATAL(RSC-016): /srv/www/pressbooks.test/current/web/app/uploads/sites/245/pressbooks/exports/jctestint-1635794782.epub/EPUB/chapter-001-headings.xhtml(19,8880): Fatal Error while parsing file: The entity "nbsp" was referenced, but not declared.
ERROR(RSC-005): /srv/www/pressbooks.test/current/web/app/uploads/sites/245/pressbooks/exports/jctestint-1635794782.epub/EPUB/chapter-001-headings.xhtml(-1,-1): Error while parsing file: The entity "nbsp" was referenced, but not declared.
ERROR(RSC-005): /srv/www/pressbooks.test/current/web/app/uploads/sites/245/pressbooks/exports/jctestint-1635794782.epub/EPUB/chapter-002-paragraph-formatting.xhtml(17,1579): Error while parsing file: value of attribute "src" is invalid; must be a URI
```
Same errors when using the exportHackathon-friday branch. So I assumed we don't introduced any additional issues in the validation process.

**Comment by richard015ar** [comment]
About styling difference between `ol` and `pre` tags in PDF was because on integrations we use DocRaptor, and locally Prince.
When using DocRaptor locally I got the same style:
![Screenshot from 2021-11-01 16-07-54](https://user-images.githubusercontent.com/13248424/139734913-9d2e3dab-ae9b-4fb0-b313-b4748a01d30e.png)
![Screenshot from 2021-11-01 16-07-49](https://user-images.githubusercontent.com/13248424/139734914-f2c1bcb7-bc40-4a58-b875-5763c0728a00.png)

**Comment by richard015ar** [comment]
Another validation issue found related to TOC:
```
ERROR(RSC-005): /srv/www/pressbooks.test/current/web/app/uploads/sites/246/pressbooks/exports/copyrightimport-1635854700.epub/EPUB/toc.xhtml(22,6): Error while parsing file: element "ol" incomplete; missing required element "li"
```
It hopefully would be fixed in #2472
